esp/esp/program/forms.py

```diff
@@ -50,16 +50,17 @@ def make_id_tuple(object_list):
 class ProgramCreationForm(BetterModelForm):
     """ Massive form for creating a new instance of a program. """
 
-    term = forms.SlugField(label='Term or year, in URL form (i.e. "2007_Fall")', widget=forms.TextInput(attrs={'size': '40'}))
-    term_friendly = forms.CharField(label='Term, in English (i.e. "Fall 07")', widget=forms.TextInput(attrs={'size': '40'}))
+    term = forms.SlugField(label='Term or year, in URL form (e.g. <tt>2007_Fall</tt>)', widget=forms.TextInput(attrs={'size': '40'}))
```

Comment:
Are you sure HTML  works here?  I would not expect it to.

Reply:
Yep, it displays correctly on the test site. I think it just carries through because it's part of a string literal then the tags get evaluated in the HTML. Not sure if there's a better/safer way to do it, though

Reply:
Ok, if it works ¯\_(ツ)_/¯

Reply:
Done

Reply:
I don't see `mark_safe()` here